patch-asy.sty

```diff
@@ -16,7 +16,7 @@
 %%
 %% Licence: GPL2+
 %%
-\ProvidesPackage{asymptote}
+\ProvidesPackage{patch-asy}
```

Comment:
No functional change, just fix a warning.
(actually why is the patch even necessary in the first place? What's the change compared to the original asymptote package?)

Reply:
the patched version numbers things in a way that allows for counters, so the figures get called e.g. Napkin-7A.asy rather than instead of Napkin-139.asy which is super brittle

Reply:
I assume by "brittle" you mean that "every figure gets recompiled every time a figure gets added somewhere in the middle of the document" (I don't think something can actually break/compiles incorrectly, just that the time overhead is huge). Sound reasonable yes.

Evidently someone would have thought of it [best practices - Is it better to use Asymptote in-document vs compiling separately? - TeX - LaTeX Stack Exchange](https://tex.stackexchange.com/questions/551253/is-it-better-to-use-asymptote-in-document-vs-compiling-separately) (that package is only made in 2014 though) although I assume there's not much gain in refactoring code at this point.

Reply:
You assume correctly.

I haven't heard of `asypictureB` before but that's good to know, maybe for future projects. Though having to run a script or enable shell-escape sounds somewhat undesirable too.